server/src/main/scala/services/crunch/deskrecs/RunnableDeskRecs.scala

```diff
@@ -48,19 +48,18 @@ object RunnableDeskRecs {
 
           bufferAsync
             .mapAsync(1) { crunchStartMillis =>
-              log.info(s"Asking for flights for ${SDate(crunchStartMillis).toISOString}")
+              log.info(s"Asking for flights for ${SDate(crunchStartMillis).toISOString()}")
               flightsToCrunch(askablePortStateActor)(portDeskRecs.minutesToCrunch, crunchStartMillis)
             }
             .map { case (crunchStartMillis, flights) =>
               val crunchEndMillis = SDate(crunchStartMillis).addMinutes(portDeskRecs.minutesToCrunch).millisSinceEpoch
               val minuteMillis = crunchStartMillis until crunchEndMillis by 60000
 
-              log.info(s"Crunching ${minuteMillis.length} minutes (${SDate(crunchStartMillis).toISOString} to ${SDate(crunchEndMillis).toISOString})")
+              log.info(s"Crunching ${flights.flightsToUpdate.size} flights, ${minuteMillis.length} minutes (${SDate(crunchStartMillis).toISOString} to ${SDate(crunchEndMillis).toISOString})")
 
               val loads = portDeskRecs.flightsToLoads(flights, crunchStartMillis)
-              val maxDesksByTerminal = deskLimitsForLoads(maxDesksProviders, flights.terminals, loads)
```

Comment:
This was filtering the terminals using the existing flights, resulting in no crunch for any terminal with no flights